mlflow/langchain/__init__.py

```diff
@@ -865,11 +896,11 @@ def load_model(model_uri, dst_path=None):
 
 @contextmanager
 def _config_path_context(code_path: Optional[str] = None):
```

Comment:
I think this breaks backwards compat right? Basically for old usecase to load the model, we need to set the old config_path as far as i can tell.
So should we create 2 contextmanager or we can update `_set_model_config` to set both the config paths?

Reply:
Let's set both for now.